apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -306,39 +306,38 @@ def get_query_server_config(name='beeswax', connector=None):
 
 def get_query_server_config_via_connector(connector):
```

Comment:
Are we changing the existing connector functionality in this method? 
I guess we also need regression testing for existing connector functionality.

Reply:
there isn't any functionality change other than some bug-fixes. The new computes is model is very similar to connectors and fits right in.